test/unit/math/prim/arr/err/is_ordered_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include <stan/math/prim/arr.hpp>
+#include <gtest/gtest.h>
+#include <limits>
+#include <string>
+#include <vector>
+
+using stan::math::is_ordered;
+
+TEST(ErrorHandling, isOrdered) {
+  std::vector<double> y_;
```

Comment:
The convention we've followed is to use a trailing `_` only for member variables in classes.  

P.S.  We also use illegal `__` on some of our generated code, but all the compilers support it.